.github/workflows/maven_release.yml

```diff
@@ -21,7 +21,6 @@ jobs:
       with:
         java-version: '11'
         distribution: 'temurin'
-        cache: maven
     - name: Initialize mandatory git config
```

Comment:
We are missing maven settings and dependency step here (or anywhere before mvn command)
  - name: Install maven dependencies
        run: ./ci/mavenDependency.sh

Reply:
Good catch, I added it.